analytics/mod.ts

```diff
@@ -0,0 +1,13 @@
+import type { App, AppContext as AC } from "deco/mod.ts";
+import manifest, { Manifest } from "./manifest.gen.ts";
+
+export type AppContext = AC<ReturnType<typeof App>>;
+
+// deno-lint-ignore no-explicit-any
+export type State = any;
```

Comment:
It can be null?

Reply:
no, but Schema will not be mergeable if we specify it as a Record<string | number, never>. set it type as {} would also be strange.